pkg/windows/service/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-windows-api-reviewers
```

Comment:
why reviewers have approvals permission? Is it intentional?

Reply:
Let's move all the edits to the OWNERS file out to a seperate PR

Reply:
I didn't expect it will be such a struggle. I am OK with the separate PR